packages/core/components/FileDetails/FileAnnotationList.tsx

```diff
@@ -94,6 +91,11 @@ export default function FileAnnotationList(props: FileAnnotationListProps) {
                 annotationValue = fileDetails.cloudPath;
             }
 
+            if (annotationValue === Annotation.MISSING_VALUE) {
```

Comment:
What is the purpose of this?

Reply:
This conditional was moved from line 74 because `annotation.extractFromFile(fileDetails);` returns MISSING_VALUE for the local file path annotation when the download is in progress (because there is not local file path yet).